source/tests/test.ts

```diff
@@ -533,6 +534,16 @@ test('is.directInstanceOf', t => {
 	t.false(m.directInstanceOf(errorSubclass, Error));
 });
 
+test('is.urlInstance', t => {
+	const url = new URL('https://google.com');
```

Comment:
http://www.example.com/ is maintained for [documentation](https://www.iana.org/domains/reserved) purposes. I recommend using them in examples, tests and documentation. 

Same for the readme.